android/quest/src/main/java/org/smartregister/fhircore/quest/util/extensions/ConfigExtensions.kt

```diff
@@ -104,7 +105,15 @@ fun List<ActionConfig>.handleClickEvent(
         }
       }
       ApplicationWorkflow.LAUNCH_REPORT -> {
-        val args = bundleOf(Pair(NavigationArg.REPORT_ID, actionConfig.id))
+        val interpolatedParams = interpolateActionParamsValue(actionConfig, resourceData)
```

Comment:
Can this interpolation be done before the `when` clause so all branches can benefit from it?